snippets/config/app/permissions.xml

```diff
@@ -1,16 +1,17 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<manifest xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="https://raw.githubusercontent.com/shopware/shopware/trunk/src/Core/Framework/App/Manifest/Schema/manifest-3.0.xsd">
-    <meta>
-        ...
-    </meta>
-    <permissions>
-        <read>product</read>
-        <create>product</create>
-        <update>product</update>
+<manifest xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+          xsi:noNamespaceSchemaLocation="https://raw.githubusercontent.com/shopware/shopware/trunk/src/Core/Framework/App/Manifest/Schema/manifest-3.0.xsd">
+  <meta>
+    ...
+  </meta>
+  <permissions>
+    <!-- CRUD shortcut - equivalent to read, create, update, delete for product (since 6.7.3.0) -->
```

Comment:
@shyim here the comment is wrong :grimacing:

Reply:
@mitelg thx, fixed :+1: 

https://github.com/shopware/docs/commit/50a83649c6d9642779b37aa1b0d17d0b5e12728d